docs/modules/ROOT/pages/kafka/kafka-reactive-binder/reactive_observability.adoc

```diff
@@ -0,0 +1,70 @@
+[[reactive-kafka-binder-observability]]
+= Observability in Reactive Kafka Binder
+
+In this section, we will describe how Micrometer based observability is enabled in the reactive Kafka binder.
+
+There is built in support for observability when it comes to producer binding, but you need to opt-in for this by enabling the following property.
+
+```
+spring.cloud.stream.kafka.binder.enable-observation
+```
+
+When this property is set to `true`, you can trace the publishing of records.
+
+Both publishing records using `StreamBridge` and regular `Supplier<?>` beans can be now traced when enabling the above property.
+
+However, on the consumer side, enabling observability is not as straightforward as on the producer side.
+
+There are two starting points for consumer binding - one a topic where the data is published via a producer binding, another one where the data is produced via not Spring Cloud Stream.
```

Comment:
`not via`?
I won't mind otherwise, just don't sounds for me.
Again: English is not my native language 😄